### PR TITLE
registering Harshit Bharani

### PR DESCRIPTION
**Pull request by HarshitBharani**
sorry  i am bit late for job challange but please accept the pull request

### CODE REVIEW

File: content/participants/ttjc@HarshitBharani/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+linkedin-link: ""
+looking-for: "job/internship"
+description: "I am a student from govt engineering college . learning react and javascript"
+---
+
+Hi! I am a Harshit Bharani let me walk you through my intro, projects and blogs.
+
+### introduction
+
+learning to code
```
Comment (gkanishk) [inline content/participants/ttjc@HarshitBharani/index.md:16]: kindly provide a proper intro

File: content/participants/ttjc@HarshitBharani/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+I know lyrics to almost all Bollywood songs.
+
+### help I need?
+
+
+
+### how to contact me?
+
+- instagram
```
Comment (gkanishk) [inline content/participants/ttjc@HarshitBharani/index.md:34]: provide links to your contact details
```
[Description](Link goes here)
```
use proper markdown.

File: content/participants/ttjc@HarshitBharani/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+
+I want to learn.
+and also want a job 
+
+## something interesting about you?
+
+I know lyrics to almost all Bollywood songs.
+
+### help I need?
```
Comment (gkanishk) [inline content/participants/ttjc@HarshitBharani/index.md:28]: if you don't need any help then mention that also

File: content/participants/ttjc@HarshitBharani/index.md (hunk 1 above)
Comment (HarshitBharani) [reply content/participants/ttjc@HarshitBharani/index.md]: updated intro

File: content/participants/ttjc@HarshitBharani/index.md (hunk 3 above)
Comment (HarshitBharani) [reply content/participants/ttjc@HarshitBharani/index.md:28]: updated need help

File: content/participants/ttjc@HarshitBharani/index.md (hunk 2 above)
Comment (HarshitBharani) [reply content/participants/ttjc@HarshitBharani/index.md]: added link for instagram and linkdin

**Pull request merged**